common/persistence/sql/sqlplugin/sqlite/plugin.go

```diff
@@ -44,6 +44,9 @@ import (
 const (
 	// PluginName is the name of the plugin
 	PluginName = "sqlite"
+	// List of non-pragma parameters
+	// Taken from https://www.sqlite.org/uri.html
+	queryParameters = "cache:immutable:mode:modeof:nolock:psow:vfs:setup"
```

Comment:
I think we should just use a map here and do exact key lookup instead of using strings.Contains.

Reply:
done